examples/synced_video.py

```diff
@@ -0,0 +1,54 @@
+"""
+Example demonstrating synced video rendering
+"""
+
+import numpy as np
+from wgpu.gui.auto import WgpuCanvas, run
+import pygfx as gfx
+
+
+canvas = WgpuCanvas(size=(512 * 3 + 100, 400))
+renderer = gfx.renderers.WgpuRenderer(canvas)
+scene = gfx.Scene()
+camera = gfx.OrthographicCamera(1800, 550)
+camera.position.y = 256
+camera.scale.y = -1
+camera.position.x = 1736 / 2
+
+colormap1 = gfx.cm.plasma
+
+
+# Just create a 512x512 random image
+def create_random_image():
+    return gfx.Image(
+        gfx.Geometry(grid=gfx.Texture(np.random.rand(512, 512).astype(np.float32) * 255, dim=2)),
+        gfx.ImageBasicMaterial(clim=(0, 255), map=colormap1),
+    )
+
+
+images = list()
+img_pos = 0
+
+
+for i in range(3):
+    images.append(create_random_image())
+    images[-1].position.x = img_pos
+    img_pos += 50 + 512
+    scene.add(images[-1])
+
+
+# update with new random image
+def update_img(obj):
+    for img in images:
+        img.geometry.grid = gfx.Texture(np.random.rand(512, 512).astype(np.float32) * 255, dim=2)
```

Comment:
Instead of creating a new Texture, you can also just update the data. I think that would improve performance.

Reply:
Thanks for the suggestion!

I tried this:
`img.geometry.grid.data[:] = np.random.rand(512, 512).astype(np.float32) * 255`

This causes the image freeze, no updating

Reply:
Ah, yes! You'd need to add this in order to let the renderer know the buffer needs to be re-uploaded to the GPu on the next frame:

`img.geometry.grid.update_range((0, 0, 0), img.geometry.grid.size)`

Reply:
Thanks! This works: `img.geometry.grid.update_range((0, 0, 0), img.geometry.grid.size)`

Performance is still the same with 3 vids, but with 6 recreating the texture gives ~10Hz, and just updating the data gives ~20Hz!

Reply:
Awesome!